resources/views/admin/createUser.blade.php

```diff
@@ -0,0 +1,187 @@
+<x-app-layout>
+    <div>
+        @include('layouts.navbar')
+    </div>
+    <div class="container">
```

Comment:
acertar aqui também posteriormente, tudo deve estar dentro de uma div container com duas rows se for o caso

Reply:
Bem observado. ;)